third_party/protobuf

```diff
@@ -1 +1 @@
-Subproject commit 644a6a1da71385e9d7a7a26b3476c93fdd71788c
```

Comment:
You probably want to revert this change from this pull request: it's very unrelated.